libres/lib/res_util/file_utils.cpp

```diff
@@ -0,0 +1,15 @@
+#include <filesystem>
+
+#include <ert/res_util/file_utils.hpp>
+
+namespace fs = std::filesystem;
+
```

Comment:
```cpp
// Header
File *mkdir_fopen_wrapper(const char *filename, const char *mode="w"); // please change name :P

// code
File *mkdir_fopen_wrapper(const char *filename, const char *mode) {
    return mkdir_fopen(fs::path(filename), mode)
}

// call
mkdir_fopen_wrapper("filename")
```

Reply:
Not a big fan of wrappers really.
We have an ongoing epic to reduce the amount of wrappers and I don't think we should add more unless strictly necessary.
We do however live in a democracy so what do you think @eivindjahren?  :)

Reply:
did not know we were reducing the use of wrappers. if that is the case. then this will defeat the purpose of that epic.

Reply:
I think we are trying to move from `const char *filename` to `fs::path` so I think its fair to not make more functionality to support the old useage. Also the `_wrapper` name is not good.